packages/shared/src/components/squads/Details.tsx

```diff
@@ -236,12 +261,40 @@ export function SquadDetails({
             }}
           />
         )}
+        {!createMode && (
+          <SquadSettingsSection title="Status" className="w-full">
+            <Radio
+              name="status"
+              options={privacyOptions}
+              value={privacy}
+              className={{ container: 'gap-4' }}
+              onChange={(value) => setPrivacy(value)}
+            />
+            {status === SquadStatus.Rejected && (
+              <Alert
+                type={AlertType.Error}
+                flexDirection="flex-row"
+                className="mt-2"
+              >
+                <p className="flex-1">
+                  You did not pass the review process. You can try again
+                  in&nbsp;{daysLeft}
+                  &nbsp;days. To increase your chances to pass in the next round
+                  you can contact us at &nbsp;
```

Comment:
I don't think this needs to be non breaking space. Normal space should be enough.
```suggestion
                  you can contact us at{' '}
```

Reply:
Yeah, either way works, was just accepting the changes the lint was suggesting.